aries-site/src/pages/whats-new.mdx

```diff
@@ -52,6 +52,85 @@ regarding either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## January 4 - January 21, 2022
+
+### "Big Things" progress updates
```

Comment:
Still wondering if this should just be "Big Things" Updates or "Big Things Progress" -- progress and updates feels redundant

Reply:
I changed this to be `### Roadmap Updates`